goal_src/jak1/engine/mods/mod-custom-code.gc

```diff
@@ -238,7 +238,7 @@
    (until (time-elapsed? current-timea (seconds 0.5))
    (suspend)
    )
-   (stop-sound-file)
+   (stop-sound-file "data/custom_assets/jak1/SND/windy.wav")
```

Comment:
not really sure how this worked before (since it was really calling `stopAllSounds`)

when I left it that way all music stopped if you left the chem fac silo, I assume you just want the windy sound to stop. didn't see `stop-sound-file` used anywhere else